firebase-firestore/ktx/src/main/kotlin/com/google/firebase/firestore/ktx/Firestore.kt

```diff
@@ -14,18 +14,28 @@
 
 package com.google.firebase.firestore.ktx
 
+import android.support.multidex.BuildConfig
```

Comment:
I understand you needed this change to build it locally, but its not the correct `BuildConfig` class. If I'm not wrong we rely on a `BuildConfig` class generated at build time under the same package name (`com.google.firebase.firestore.ktx`) which is why we don't have an import for it.

Can you please remove this import line?

Reply:
Good catch, that's done ✅  Turns out it wasn't even required for building.